src/main/resources/com/google/api/codegen/csharp/gapic_snippets.snip

```diff
@@ -226,6 +228,30 @@
 @end
 
 # snippetMethod: StaticLangApiMethodSnippetView
+@private streamingRequestObjectMethodServer(snippetMethod)
+    public async Task {@snippetMethod.snippetMethodName}()
+    {
+        // Snippet: {@snippetMethod.method.name}({@snippetMethod.method.serviceRequestTypeName},CallSettings)
+        // Create client
+        {@snippetMethod.apiClassName} {@snippetMethod.apiVariableName} = {@snippetMethod.apiClassName}.Create();
+        // Initialize request argument
+        {@topLevelInitCode(snippetMethod.method.initCode)}
+        // Make the request, returning a streaming response
```

Comment:
The terminology and variable names here are a bit confusing, but I suspect that's inherent. I haven't come up with anything better.